### PR TITLE
kommander2: Manually sync latest kommander 2.0 docs

### PR DESCRIPTION
**Pull request by gracedo**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

**Review by karishmagadiya — APPROVED** [review]
approved the changes

**Review by takirala — APPROVED** [review]

**Comment by gracedo** [comment]
Can preview at http://docs-d2iq-com-pr-3663.s3-website-us-west-2.amazonaws.com/dkp/kommander/2.0/install/ (konvoy links still broken here because konvoy 2.0 docs are hidden in `main`)

**Review by GoelDeepak — APPROVED** [review]

File: pages/dkp/kommander/2.0/index.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ title: Kommander 2.0
 version: 2.0
 menuWeight: 0
 subtree:
-beta: true
+  beta: true
```
Comment (msdolbey) [inline pages/dkp/kommander/2.0/index.md:8]: formatting
```suggestion
beta: true
```

File: pages/dkp/kommander/2.0/index.md [hunk 2]
```diff
@@ -5,7 +5,8 @@ title: Kommander 2.0
...
+  draft: true
```
Comment (msdolbey) [inline pages/dkp/kommander/2.0/index.md:9]: formatting
```suggestion
draft: true
```

File: pages/dkp/kommander/2.0/index.md (hunk 1 above)
Comment (gracedo) [reply pages/dkp/kommander/2.0/index.md:8]: looking at https://github.com/mesosphere/dcos-docs-site/pull/3578/files I think these should be indented under `subtree:` ?

**Review by msdolbey — APPROVED** [review]
👍 
Manually

File: pages/dkp/kommander/2.0/index.md (hunk 1 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.0/index.md:8]: You're right, you're right

**Pull request merged**